### ISSUE TITLE
0.12.0 styles/scripts 404

### ISSUE THREAD

**Issue by rudeyak** [kalvn/Shaarli-Material#116]
With the new release, shaarli running in a subdirectory (https://my.doma.in/Sha12/.......) I get 404 errors for styles.min.css and scripts.min.js, e.g.

https://my.doma.in/Sha12/tpl/material/dist/styles.min.css?v=85a1f6706320a0d7e17e6a159873fbd3e4b00ae0c3c15f1170669a2d0aeface6

```
2020/10/13 08:28:19 [error] 13162#13162: *5727 FastCGI sent in stderr: "PHP message: PHP Notice:  Undefined variable: _PAGE_ in /var/www/Sha12/tmp/page.header.c04e6db3e3da5484d981ebaa00b75abc.rtpl.php on line 3" while reading response header from upstream, client: 65.29.xx.xx, server: my.doma.in, request: "GET /Sha12/admin/configure HTTP/2.0", upstream: "fastcgi://unix:/var/run/php/php7.2-fpm.sock:", host: "my.doma.in", referrer: "https://my.doma.in/Sha12/admin/configure"
2020/10/13 08:28:19 [error] 13162#13162: *5727 FastCGI sent in stderr: "PHP message: PHP Notice:  Undefined variable: _PAGE_ in /var/www/Sha12/tmp/page.header.c04e6db3e3da5484d981ebaa00b75abc.rtpl.php on line 3" while reading response header from upstream, client: 65.29.xx.xx, server: my.doma.in, request: "GET /Sha12/tpl/material/dist/styles.min.css?v=85a1f6706320a0d7e17e6a159873fbd3e4b00ae0c3c15f1170669a2d0aeface6 HTTP/2.0", upstream: "fastcgi://unix:/var/run/php/php7.2-fpm.sock:", host: "my.doma.in", referrer: "https://my.doma.in/Sha12/admin/configure"
2020/10/13 08:28:20 [error] 13162#13162: *5727 FastCGI sent in stderr: "PHP message: PHP Notice:  Undefined variable: _PAGE_ in /var/www/Sha12/tmp/page.header.c04e6db3e3da5484d981ebaa00b75abc.rtpl.php on line 3" while reading response header from upstream, client: 65.29.xx.xx, server: my.doma.in, request: "GET /Sha12/tpl/material/dist/scripts.min.js?v=85a1f6706320a0d7e17e6a159873fbd3e4b00ae0c3c15f1170669a2d0aeface6 HTTP/2.0", upstream: "fastcgi://unix:/var/run/php/php7.2-fpm.sock:", host: "my.doma.in", referrer: "https://my.doma.in/Sha12/admin/configure"

```

**Comment by kalvn** [comment]
I think you have to update this settings in Shaarli configuration:

![image](https://user-images.githubusercontent.com/6149640/96172013-0b701200-0f26-11eb-9509-6761c24d0c80.png)

Do you have the same issue with the default theme?

**Comment by rudeyak** [comment]
Hmmm. No, default and vintage themes work OK (with `header_link` set to `"\/Sha12\/"` under `general:` in the config). The installer _seemed_ to point to the right URL as of 0.12.0.

**Comment by kalvn** [comment]
What version of Shaarli Material are you using? I just released v0.12.0, can you please give a try with this one?

**Comment by rudeyak** [comment]
Just tried with 0.12.0 (and shaarli 0.12.1), no errors! 🥳
